src/main/java/spoon/support/reflect/CtExtendedModifier.java

```diff
@@ -93,4 +93,11 @@ public ElementSourceFragment getOriginalSourceFragment() {
 			return ElementSourceFragment.NO_SOURCE_FRAGMENT;
 		}
 	}
+
+	@Override
+	public String toString() {
+		return "CtExtendedModifier{"
+				+ "implicit=" + implicit
+				+ ", kind=" + kind + '}';
+	}
```

Comment:
Failing tests where modifiers don't match were hard (impossible) to read. I can move this change to the bug fixing PR too, there it would be more related probably.

Reply:
Right, I see. That's a fine enough justification for that.